src/overlays/actors/ovl_En_Dnq/z_en_dnq.c

```diff
@@ -459,6 +459,12 @@ void EnDnq_HandleCutscene(EnDnq* this, PlayState* play) {
         DEKU_KING_ANIM_IDLE,          DEKU_KING_ANIM_IDLE_MORPH,
         DEKU_KING_ANIM_SURPRISE,      DEKU_KING_ANIM_JUMPED_ON_START,
         DEKU_KING_ANIM_JUMPED_ON_END, DEKU_KING_ANIM_JUMPED_ON_END_MORPH,
+#ifdef AVOID_UB
+        //! @bug Z2_DEKU_KINGCutsceneData_008258 provides a cue id of 6, reading out-of-bounds
+        //! of this array that happens to be 0 padding.
+        // Avoid UB: Add an explicit array member rather than relying on 0 padding.
+        DEKU_KING_ANIM_IDLE,
+#endif
```

Comment:
should this mention mm3d and how it proves that it was infact an original bug?

Reply:
i would be interested to see how MM3D proves this to be padding, because it does currently match without the AVOID_UB tag

Reply:
see discord (gcc thread)